content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -372,6 +374,16 @@ implemented using the NetworkPolicy API.
 - The ability to prevent loopback or incoming host traffic (Pods cannot currently block localhost
   access, nor do they have the ability to block access from their resident node).
 
+## NetworkPolicy's impact on existing connections
+
+When the set of NetworkPolicies that applies to an existing connection changes - this could happen
+either due to a change in NetworkPolicies or if the relevant labels of the namespaces/pods selected by the
+policy (both subject and peers) are changed in the middle of an existing connection - it is
+implementation defined as to whether the change will take effect for that existing connection or not.
+Example: A policy is created that leads to denying a previously allowed connection, the underlying CNI
```

Comment:
```suggestion
Example: A policy is created that leads to denying a previously allowed connection, the underlying network plugin
```

see line 45

Reply:
ah ack, good point, will repush this asap

Reply:
done.